### ISSUE TITLE
Flame-retardant vodka

### ISSUE THREAD

**Issue by serin-delaunay** [Attnam/ivan#121]
If I have a burning object in my inventory, and dip it into a bottle of vodka, the flames are doused just as if I had dipped it into any other liquid. I don't have much personal experience combining flames and alcohol, but in IVAN, where vodka bottles can explode at the drop of a hat, it seems strange that they can be used for this purpose without mishap.

**Comment by fejoa** [comment]
This is true, and that would be because it hasn't been coded in yet. It is behaviour that is part of the fire subsystem. I'll try to look at this again at some point. Thanks for spotting the inconsistency, and please do put up more of your observations and bug identifications because it really helps us to know what to fix :)

**Comment by red-kangaroo** [comment]
Flammable (vodka) or hot (lava) liquids no longer douse flames.
